R/viz.R

```diff
@@ -2781,15 +2794,36 @@ fancy_sv_chrdistplot = function(this_sample,
 
   #get maf data for a specific sample.
   if(missing(maf_data) && is.null(maf_path)){
-    maf = assign_cn_to_ssm(this_sample = this_sample, coding_only = coding_only, from_flatfile = from_flatfile, use_augmented_maf = use_augmented_maf)$maf
+    if(ssm){
+      maf = assign_cn_to_ssm(this_sample = this_sample, coding_only = coding_only, from_flatfile = from_flatfile, use_augmented_maf = use_augmented_maf)$maf
+    }else{
+      maf = get_combined_sv(sample_ids = this_sample, projection = projection, min_vaf = min_vaf) %>%
+        dplyr::select(CHROM_A, START_A, END_A, manta_name)
+
+      #get manta results in required format
+      maf = data.frame(maf$CHROM_A, maf$START_A, maf$END_A, do.call(rbind, strsplit(maf$manta_name, split = ":", fixed = TRUE)))
+
+      #rename variables
+      names(maf)[1] = "Chromosome"
+      names(maf)[2] = "Start_Position"
+      names(maf)[3] = "End_Position"
+      names(maf)[4] = "Variant_Type"
+
+      #filter out translocations and set order of variables
+      maf = dplyr::filter(maf, Variant_Type %in% c("MantaDEL", "MantaDUP")) %>%
```

Comment:
This will probably drop SVs coming not from manta, but other sources (like GRIDSS)?

Reply:
The non-Manta ones may not have annotations in that column. Leave it for now. This can be a TO DO